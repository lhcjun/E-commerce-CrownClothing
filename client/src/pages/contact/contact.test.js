import React from 'react';
import { shallow } from 'enzyme';
import ContactPage from './contact.component';

it('should render ContactPage component', () => {
    expect(shallow(<ContactPage />)).toMatchSnapshot();
});