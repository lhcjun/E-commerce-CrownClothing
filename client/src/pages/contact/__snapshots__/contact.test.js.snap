// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ContactPage component 1`] = `ShallowWrapper {}`;
